main/commons/src/main/java/org/cryptomator/common/settings/PwBackend.java

```diff
@@ -0,0 +1,27 @@
+package org.cryptomator.common.settings;
+
+import org.apache.commons.lang3.SystemUtils;
+
+public enum PwBackend {
```

Comment:
I'd prefer another name like

```suggestion
public enum KeychainBackend {
```